portfolio/src/main/webapp/index.html

```diff
@@ -8,21 +8,25 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>CJ's Portfolio</h1>
       <a href="https://github.com/cjhbh3" target="_blank">Github</a>
+      <h2>About Me</h2>
       <p>
-          Hi, my name is CJ. I just finished my freshmen year at Missour S&T.
-          I was born in Indiana and now live in Missouri. 
-          I have worked in many IT jobs.
-          My love for computer programming begain freshmen year of high school
-          when I became the lead programmer for my high school's FTC Robotics team.
+        Hi, my name is CJ.</br>
+        I just finished my freshmen year at Missour S&T.</br>
+        I was born in Indiana and now live in Missouri.</br>
+        I have worked in many IT jobs.</br>
+        Why did I start CS? My experience as a lead programmer in FTC Robotics during High School.</br>
       </p>
       <p>This is my dog, Juliette.</p>
       <img src="/images/Juliette.jpg" width="500" height="500">
       <p>Here is what I look like!</p>
       <img src="/images/SelfPic.JPG" with="500" height = "500">
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Fetching Text</p>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch Me that Text</button>
+      <div id="quote-container"></div>
+      <p>Click here for one of my favorite tv show quotes:</p>
+      <button onclick="addRandomGreeting()">Give me a Quote</button>
```

Comment:
How is quote vs greeting is being used here?

Reply:
The quote fetches the text that is located on the /data page, while the greeting is randomly generated quote of four potential. I can update the function names to reflect these better.